src/test/python/apache/aurora/config/test_loader.py

```diff
@@ -93,6 +101,44 @@ def test_load_json_single():
   assert new_job == job
 
 
+def test_load_json_memoized():
+  """
+  This test reads 2 jobs from the MESOS_CONFIG_MULTI string.
+  It then tests memoization by:
+  1. write job[0] as json to tmp/config.json
+  2. load_json(tmp/config.json, is_memozied=True) to prime the cache
+  3. overwrite tmp/config.json with job[1] as json.
+  4. call load_json(tmp/config.json, is_memoized=True) and verify that job[0] is read
```

Comment:
I am confused. So we are returning the same configuration when the file's content has changed because the filename's match?

Reply:
We memoize on the filename

Reply:
The file should not be written during the CLI evaluation so I'm not sure if that concern is valid.

Reply:
Ive modified both the code and the tests to be cached on filecontent instead of filename.

Reply:
Thank you.